Cargo.toml

```diff
@@ -31,12 +32,11 @@ phf = { version = "0.9.0", features = ["macros"] }
 # ID / key generation
 ulid = "0.4.1"
 nanoid = "0.4.0"
-base64 = "0.13.0"
 
 # serde
 serde_json = "1.0.57"
 serde = { version = "1.0.115", features = ["derive"] }
-validator = { version = "0.11", features = ["derive"] }
+validator = { version = "0.14", features = ["derive"] }
 rmp-serde = { git = "https://github.com/insertish/msgpack-rust", rev = "5bf2c24203ad422233cf35b7b7bfad9f7e811814" }
```

Comment:
Is there a reason you have this pinned to an older version instead of `"1.1"`?
It's causing `cargo vendor` conflicts.